### PR TITLE
added an update example

### PR DESCRIPTION
**Pull request by hermesxgjini**

### CODE REVIEW

File: docs/content/core/crud.fsx [hunk 1]
```diff
@@ -121,6 +121,27 @@ mvps2
 
 ctx.SubmitUpdates()
 
+(** update a single row 
+*)
+
+type Employee2 = {
```
Comment (Thorium) [inline docs/content/core/crud.fsx]: You don't need this record to anything

File: docs/content/core/crud.fsx [hunk 2]
```diff
@@ -121,6 +121,27 @@ mvps2
...
+    LastName:string
+}
+
+let updateEmployee (employee: Employee2) =
+    query {
+        for p in ctx.Public.Employee2 do
+        where (p.Id = employee.Id)
+    }
+    |> Seq.iter( fun e ->
+        e.FirstName <- post.FirstName
```
Comment (Thorium) [inline docs/content/core/crud.fsx]: What is post?

File: docs/content/core/crud.fsx (hunk 1 above)
Comment (hermesxgjini) [inline docs/content/core/crud.fsx]: I added for the updateEmployee function because the Employee record above didn`t have an Id field

**Comment by Thorium** [comment]
Thanks!

**Pull request merged**